src/snowflakes/search_views.py

```diff
@@ -36,7 +38,8 @@ def includeme(config):
     config.add_route('report', '/report{slash:/?}')
     config.add_route('matrixv2_raw', '/matrixv2_raw{slash:/?}')
```

Comment:
Do we still need this?

Reply:
Yes, most importantly these are demo implementations for others to learn from (and to remind myself in the future). Some of them might be useful.